### ISSUE TITLE
[3.2] Templating for exception controller

### ISSUE THREAD

**Issue by GawainLynch** [bolt/bolt#5714]
This can be rendered, and we're passing to much in to the renderer, so we need to clean up both the UI and the requirement on twig variables we don't use/want/need

![image](https://cloud.githubusercontent.com/assets/1427081/18059758/c2dfa128-6e1a-11e6-896c-8707c37f47ce.png)

**Comment by GawainLynch** [comment]
Per discussion in Slack … passing back over to you @bobdenotter

**Comment by GawainLynch** [comment]
We don't know when in the execution cycle the exception is going to happen/be handled, except that for exception we render (i.e. this issue).

Our default backend Twig rendering passes in everything, including the proverbial kitchen sink, in terms of object & services, and if some of those are not booted/registered/working then we can't render the *actual* exception … instead of 
```
Parameter 3 passed to `Bolt\Foo\Bar` must be of type `Bolt\Koala`, `Bolt\DropBear provided`
```

We could end up with the above Twig exception, such as the one above. In terms of getting to be bottom of a problem, it introduces an artificial delay.

**Comment by bobdenotter** [comment]
Fixed by #5869, until proven otherwise.
